taegis_magic/commands/tenant_profiles.py

```diff
@@ -597,3 +612,191 @@ def mfa_remove(
     )
 
     return normalized_results
+
+
+@network_template.command("generate")
+@tracing
+def network_template_generate(
+    filename: Annotated[
+        Path,
+        typer.Option(
+            exists=False,
+            file_okay=True,
+            dir_okay=False,
+            readable=False,
+            writable=True,
+            resolve_path=True,
+            callback=verify_file,
+        ),
+    ] = Path("taegis_network_range_template.xlsx"),
+):
+    """Generate a template file for tenant profiles network ranges."""
+
+    input_ = NetworkRangeCreateMtpInput(
+        cidr="192.0.2.0/24",
+        description="RFC 5737 TEST-NET-1",
+        network_type=MtpNetworkType.OTHER,
+        is_critical=False,
+    )
+    df = pd.json_normalize([asdict(input_)], max_level=3)
+
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore")
+        df = df.applymap(excel_normalize, na_action="ignore")
+
+    df.to_excel(filename, index=False)
```

Comment:
The excel writer is an optional `pandas` dependency that is not guaranteed to be present based on our current `install_requires`. We can use CSV instead, catch the `ModuleNotFound` error, or add one of the python XLSX packages to our list of dependencies.

Reply:
added